src/tv2-common/__tests__/jinglePartProperties.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { GetJinglePartPropertiesFromTableValue } from 'tv2-common'
+
+describe('GetJinglePartPropertiesFromTableValue', () => {
+	it('Calculates values correctly', () => {
```

Comment:
A good naming convention for tests is "methodName_inGivenState_expectResult".
Examples of this could be:
`fetchBlueprints_noBluePrintsAvailable_returnEmptyArray` or `multiply_twoAndFour_returnEight`.

When using describes we can alter that convention to put the method name in the describes (as you did here).

The point is, that we want to be able to figure out what is wrong (or have a pretty good idea) just by looking the name of the failing test, as that is the very first thing we see when a test fails and will often give us a pretty good idea of `what` is going wrong instead of just telling us that `something` is going wrong.
(And when you combine the information from the well named test with the error message of the assertion, you get loads of valuable information for free)